quadratic/sgp_test.go

```diff
@@ -34,9 +34,6 @@ func TestSGP(t *testing.T) {
 	if err != nil {
 		t.Fatalf("error when generating random matrix: %v", err)
 	}
-	// FIXME tests sometimes fail when negative values are present in the matrix F!
-	//f[0][0].Set(big.NewInt(-99))
-	//f[1][0].Set(big.NewInt(-99))
```

Comment:
Does this no longer occur? Or is there another reason why it was deleted?

Reply:
Yes, I believe it was fixed.